covid_graphs/covid_graphs/prepare_data.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
```

Comment:
These lines can be removed in all files that have been converted.

Reply:
Done